tools/src/main/scala/au/com/cba/omnia/ebenezer/fs/Glob.scala

```
@@ -25,6 +24,10 @@ object Glob {
 
   def paths(conf: Configuration, paths: List[Path]): List[Path] = {
     val fs = FileSystem.get(conf)
-    paths.flatMap(path => fs.globStatus(path).toList.map(_.getPath))
+    paths.flatMap(path => Option(fs.globStatus(path))
```

If an invalid path is provided, It is null. I would have expected hadoop to throw `Invalid Path Exception` or so. But it just returns null. (Which is apparently the behaviour)

http://grepcode.com/file/repository.cloudera.com/content/repositories/releases/org.apache.hadoop/hadoop-common/2.5.0-cdh5.3.2/org/apache/hadoop/fs/Globber.java#286